.github/workflows/codeql-analysis.yml

```diff
@@ -13,10 +13,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, 1.x ]
```

Comment:
If you make a draft PR from `1.x` to `master` (after merging), all the workflows would run and this is not necessary.
So you can either keep this and later remove the 1.x from workflows, or merge without adding the 1.x workflow but open a draft PR to check that everything is green.